index.html

```diff
@@ -1,289 +1,307 @@
-                    <a href="#Contact">contact</a>
-                </div>
-                <!--hamburguer tap-->
+        <!-- header navbar -->
+        <!--top nav -->
+        <div class="topnav">
+            <!--hamburhuer tap-->
 
-                <nav>
-                    <a href="javascript:void(0);" class="icon" onclick="myFunction()">
-                        <i class="fa fa-bars"></i>
-                    </a>
-                </nav>
 
+            <a href="https://www.linkedin.com/in/roger-smith-a35738179/" class="active">John Doe</a>
+
+            <div id="myLinks">
+                <a href="#box1">Portafolio</a>
+                <a href="#about">About</a>
+                <a href="#formato">Contact</a>
             </div>
-        
+            <!--hamburguer tap-->
+
+            <nav>
+                <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+                    <i class="fa fa-bars"></i>
+                </a>
+            </nav>
+
+        </div>
+
     </header>
 
     <section>
         <div class="roger">
-                    <div class="pad">
-                    <h1>Hey There.</h1>
-                    <h2 class="k">I´m Roger</h2>
-                </div>
-            
+            <div class="pad">
+                <h1>Hey There.</h1>
+                <h2 class="k">I´m Roger</h2>
+            </div>
+
             <div class="supporttest">
                 <p class="developer">I´am a Software Developer</p>
                 <div class="theme1">
-                    <p class="text2">I can help you build a product , feature or website Look through some of my work
-                        and
-                        experience! If you like what you see and have a project you need coded, don’t hestiate to
-                        contact me. </p>
+                    <p class="text2">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me. </p>
                 </div>
             </div>
-            
-            <ul class="palet" >
-                <li><a href="#"><img class="gith" src="pictures/github.png" alt="github icon"></a></li>
-                <li> <a href="#"><img src="pictures/linkedin.png" alt="github icon"></a></li>
-                <li><a href="#"><img src="pictures/Enabled.png" alt="github icon"></a></li>
-                <li><a href="#"><img src="pictures/Disabled.png" alt="github icon"></a></li>
-                <li> <a href="#"><img src="pictures/magic.png" alt="github icon"></a></li>
+
+            <ul class="palet">
+                <li>
+                    <a href="https://github.com/latinogan"><img class="gith" src="pictures/github.png" alt="githubaccount"></a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/roger-smith-a35738179/"><img src="pictures/linkedin.png" alt="Linkedinaccount"></a>
```

Comment:
- [ ] Kindly add the alt like "Linkedin account". Make sure to leave a space between the words so that when a disable turns on the listening mode the image alternative can be heard correctly. Make sure to change all the alt image the same way as this.

Reply:
Thank you for the CR